pino/pino-tests.ts

```diff
@@ -1,6 +1,8 @@
 // This contains all of the code from the docs on https://www.npmjs.com/package/pino
 
-import PINO = require('pino')
+import * as PINO from 'pino';
```

Comment:
Since this behaves as a CommonJS module, you should leave the CommonJS-specific syntax. If this were compiled to ES6  then `PINO` would not be callable if you use `import * as PINO from 'pino';`.